files/__addonLocation__/gitignore

```diff
@@ -3,3 +3,8 @@
 # will also appear in published NPM packages.
 /README.md
 /LICENSE.md
+# Build output
+/dist/
+/declarations/
+# npm/pnpm/yarn pack output
+*.tgz
```

Comment:
@NullVoxPopuli Can you explain to me how you had encountered the need to ignore `*.tgz`?

When I ran the `publish --dry-run` command, I didn't see any file with the `.tgz` extension so I feel like we wouldn't need lines 9-10.

Reply:
If you run pnpm pack (for example, to verify a package would publish as you expect), you get a tgz file in your addon directory - we don't want that comitted.

Reply:
Ah, ok. Thanks for providing more context.